_raml/apps/products/products.raml

```diff
@@ -102,14 +102,15 @@ post:
                 - /productLength<br>
                 - /productWidth<br>
                 - /productHeight<br>
+                - /producttype³<br>
                 - /priceInfo/taxClass/taxClassId³<br>
                 - /productImage⁴<br>
                 - /stocklevel²³<br>
```

Comment:
Short comment on that, maybe we should separate the superscripts 2 and 3 with a comma?

Reply:
I think  ²,³ is not looking fine.   It's clear if we separate by space